Articles of Association.md

```diff
@@ -0,0 +1,66 @@
+1. Name
+    1. The name of the association is "Feather Association"
+2. Purpose
+    1. The purpose of the association is
+        1. The promotion of feather
+        2. The development of feather
+        3. The distribution of feather and associated software and tools
+3. Membership
+    1. Anyone can be admitted as a Paying Member.
+    2. Registration takes place on the association's registration form. The membership becomes valid once the Paying Member has paid the membership fee. The membership lasts for one financial year from the date of payment.
+    3. Paying Members pay a membership fee determined by the general meeting.
+4. General Assembly
+    1. The general meeting is the highest authority of the association.
+    2. The Annual General Meeting is held once a year before the end of the ______ month and is convened with at least three weeks' notice by email to members who are eligible to vote.
+    3. The public is eligible to attend the general meeting.
+    4. Voting Members are eligible to vote. Voting by proxy is not possible.
+    5. The agenda for the annual General Assembly must contain at least the following items:
+        1. Election of secretary/transcriber
+        2. Election of vote teller
+        3. Election of chairman of the meeting
+        4. Financial reporting
+        5. Approval of new Voting Members by voting
+        5. Consideration of proposals received
+        6. Determination of membership fee for Paying Members
+        7. Approval of the budget
+        8. Election of chairman of the Board of Directors
+        9. Election of deputy chairman of the Board of Directors
+        10. Election of 3-7 board members and 2 deputy members
+        11. Election of auditor (s) and deputy auditor 
+        12. TODO: Optionally/Any subject not on the agenda
+    6. Proposals that are to be considered at the General Assembly must be received by the board of directors no later than 8 days before the General Assembly. Proposals for amendments to the articles of association that are to be considered must be received by the Board of Directors no later than three weeks before the General Assembly.
+    7. The General Assembly is chaired by a chairman who cannot be a member of the board.
+    8. The General Assembly makes its resolutions by 1/2 of votes and by public written voting.
```

Comment:
`public written voting` => `Votes are public`? Or is that not what this means?

Reply:
Votes could be done by raise of hands... which for obvious reason would be hard. That's why I specified that votes I cast by writing.